public/index.js

```diff
@@ -23,4 +24,32 @@
     passwordString = '/search/' + encodeURIComponent(e.target.value);
     serverRequest(passwordString, renderDOM);
   });
-}());
+
+  form.addEventListener('submit', function (e) {
+    e.preventDefault();
+    var options = document.getElementById('password-examples').childNodes;
```

Comment:
You can shorten this to dataList as it is already declared in the top of the file